source/dsl/DangerUtilsDSL.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * The Danger Utils DSL contains utility functions
+ * that are specific to universal Danger use-cases.
+ */
+export interface DangerUtilsDSL {
+
+  /**
+   * Creates an HTML link.
+   *
+   * @param {string} href the HTML link's desintation
+   * @param {string} text the HTML link's text
+   * @returns {string} the HTML <a> tag
+   */
+  href(href: string, text: string): string
+
+  /**
+   * Converts an array of strings into a sentence.
+   *
+   * @param {Array<string>} array the array of strings
+   * @returns {string} the sentence
+   */
+  sentence(array: Array<string>): string
```

Comment:
haha, the objective-c person in me is screaming "call this toSentence" 

```js
import { utils } from 'danger'
const results = utils.sentence(problems)
```

Yeah, I guess this feels about right